src/status_im/chat/models/input.cljs

```diff
@@ -22,12 +22,9 @@
                      (aget emoji-map "char")
                      original)))))
 
-(defn text-ends-with-space?
-  "Returns true if the last symbol of `text` is space"
-  [text]
-  (when text
-    (= (str/last-index-of text const/spacing-char)
-       (dec (count text)))))
+(defn text-ends-with-space? [text]
+  (and (not (nil? text))
```

Comment:
This can be rewritten as 
```clojure
((fnil str/ends-with? "") text const/spacing-char)
```
but only if this makes code more readable, of course.